pkg/distro/fedora/distro.go

```diff
@@ -444,6 +444,8 @@ var (
 			},
 			// Unset hostname so it isn't written (necessary for systemd-firstboot).
 			Hostname: common.ToPtr(""),
+			// Unset locale so it isn't written (necessary for systemd-firstboot).
+			Locale: common.ToPtr(""),
```

Comment:
Same as previous comment about hostname.

Setting this to `ptr("")` will make the stage run and write `""`, no?

Reply:
Setting it to `nil` does the wrong thing when the merge between image configs gets done; it then gets turned into the distro default.

The `os` pipeline verifies that the value is non-empty (e.g. `if p.Language != ""`) as a conditional for writing the stage so it doesn't add the stage at all.